clients/fides-js/src/components/fides.css

```diff
@@ -477,6 +481,28 @@ div#fides-consent-content .fides-modal-description {
   justify-content: center;
 }
 
+.fides-modal-secondary-actions .fides-brand {
+  position: absolute;
+  right: var(--fides-overlay-padding);
+  bottom: var(--fides-overlay-padding);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: var(--fides-overlay-font-size-body-xs);
+}
+
+.fides-modal-secondary-actions .fides-brand-link {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+  text-decoration: none;
+  color: var(--fides-overlay-font-color);
+}
+
+#ethyca-logo {
```

Comment:
super-nit: this CSS rule is global. `fides.css` is installed on lots of customer websites, some of which might have their own reasons to include a div with an id like `#ethyca-logo`... can we adjust this to be less likely to conflict? Let's use a class name and a selector that scopes to our own div.